src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -710,6 +712,18 @@ private async Task<bool> ValidateInputAsync(Message message, State state, LazyIn
             return true;
         }
 
+        private JObject AddStateIdToSettings(string actionType, JObject jObjectSettings, string stateId)
```

Comment:
this method doesn't need to return a value...

Reply:
ok